string.c

```diff
@@ -11295,12 +11295,12 @@ rb_str_unicode_normalized_p(int argc, VALUE *argv, VALUE str)
  *
  * == What's Here
  *
- * First, what's elsewhere. \Class \Symbol:
```

Comment:
This is the documentation for `Symbol` right? I'm not a fan of the class linking to itself.

Reply:
Fixed (throughout class Symbol).